doc/nrf/links.txt

```diff
@@ -98,6 +98,8 @@
 .. _`nRF Connect platform overview`: https://github.com/nrfconnect/sdk-connectedhomeip/blob/5b90f5/docs/guides/nrfconnect_platform_overview.md
 .. _`other controller setups`: https://github.com/nrfconnect/sdk-connectedhomeip/tree/5b90f5/src/controller
 .. _`ZCL Advanced Platform`: https://github.com/project-chip/zap
+.. _`Matter nRF Connect releases`: https://github.com/nrfconnect/sdk-connectedhomeip/releases
+.. _`Building Android CHIPTool`: https://github.com/project-chip/connectedhomeip/blob/5b90f5/docs/guides/android_chiptool_building.md
```

Comment:
I think the commit SHA in links is not up to date and you should try rebasing it. The current one seems to be 27bce6d

Reply:
Right, this was added before your PR was merged. Thanks!